### PR TITLE
Update modulesync_config [0d59329]

### PR DESCRIPTION
**Pull request by DavidS**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,9 +19,12 @@ matrix:
     script: bundle exec rake beaker
     services: docker
     sudo: required
+  - rvm: 2.3.1
+    bundler_args: --without system_tests
+    env: PUPPET_GEM_VERSION="~> 4.0"
   - rvm: 2.1.9
     bundler_args: --without system_tests
-    env: PUPPET_GEM_VERSION="~> 4.0" STRICT_VARIABLES="yes"
```
Comment (tphoney) [inline .travis.yml]: should this be removed ?

Comment (DavidS) [inline .travis.yml]: https://github.com/puppetlabs/puppetlabs-tftp/pull/37#discussion_r76256040 :-)

File: Rakefile [hunk 2]
```diff
@@ -2,13 +2,9 @@ require 'puppet_blacksmith/rake_tasks'
 require 'puppet-lint/tasks/puppet-lint'
 require 'puppetlabs_spec_helper/rake_tasks'
 
-PuppetLint.configuration.fail_on_warnings = true
 PuppetLint.configuration.send('relative')
-PuppetLint.configuration.send('disable_140chars')
```
Comment (bmjen) [inline Rakefile]: Are these puppet-lint settings being set by PSH now?

Comment (DavidS) [inline Rakefile]: Yes.

**Pull request merged**